test/unit/android-common-specs.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var rewire = require('rewire')
+  , androidCommon = rewire('../../lib/devices/android/android-common.js')
+  , chai = require('chai');
+
+chai.should();
+
+var java_version_stderr = 'Picked up _JAVA_OPTIONS: -Djava.awt.headless=true\njava version "1.8.0_22"';
+
+var fakeExec = function (_, fn) {
+  fn(null, null, java_version_stderr);
+};
+
+var _test_double = function () {
```

Comment:
BTW, we can add a .jscsrc rule to enforce this, but currently it break too many places:
`"requireCamelCaseOrUpperCaseIdentifiers": true`